Master RMarkdown Document & Render Code/excel_output.r

```diff
@@ -155,5 +160,8 @@ for (HSCP in hscp_list) {
 
 
   # Save the workbook to a file
-  openxlsx::saveWorkbook(wb, paste0(lp_path, "background data/", HSCP, ".xlsx"), overwrite = TRUE)
+  openxlsx::saveWorkbook(wb, paste0(lp_path, "Master RMarkdown Document & Render Code/Output/background data/", HSCP, " - Locality Profile data.xlsx"), overwrite = TRUE)
```

Comment:
This isn't the path / dir I was thinking about, but I like it better!

Just a note to myself to update the SDC PR (#98) to use the same path. - @JosieSteele could you review the code on that one please? Clive has reviewed the outputs for it (and this one)